### PR TITLE
chore(Makefile/buildscript): Clean up traces of travis workflow from makefile and buildscript

### PR DESCRIPTION
**Pull request by niladrih**
Signed-off-by: Niladri Halder <niladri.halder@mayadata.io>

Changes:
- Does not export few variables as it is not required anymore.
- Buildx workflow uses the same variables as used for host OS/arch build.
- removed generation of BUILDMETA from buildscript
- Deleted BUILDMETA and VERSION files.

Also in this PR:
- Created pkg/version/version.go to track version and latest GitCommit hash.

### CODE REVIEW

File: Makefile.buildx.mk [hunk 1]
```diff
@@ -48,7 +42,7 @@ docker.buildx:
 	@echo
 
 .PHONY: docker.buildx.provisioner-localpv
-docker.buildx.provisioner-localpv: DOCKERX_IMAGE_NAME=$(DOCKERX_IMAGE_PROVISIONER_LOCALPV)
+docker.buildx.provisioner-localpv: DOCKERX_IMAGE_NAME=$(PROVISIONER_LOCALPV_IMAGE_TAG)
```
Comment (akhilerm) [inline Makefile.buildx.mk:45]: :+1:

Comment (akhilerm) [reply Makefile.buildx.mk:45]: @niladrih I think this change may be needed in other repos too, I have seen the `DOCKERX_` variable used in multiple repositories. We need to change all that to the above format.

File: buildscripts/build.sh [hunk 2]
```diff
@@ -108,9 +103,9 @@ if [ $GOOS = "windows" ]; then
 fi
 
 env GOOS=$GOOS GOARCH=$GOARCH go build ${BUILD_TAG} -ldflags \
-    "-X github.com/openebs/maya/pkg/version.GitCommit=${GIT_COMMIT} \
+    "-X github.com/openebs/dynamic-localpv-provisioner/pkg/version.GitCommit=${GIT_COMMIT} \
     -X main.CtlName='${CTLNAME}' \
-    -X github.com/openebs/maya/pkg/version.Version=${VERSION}" \
+    -X github.com/openebs/dynamic-localpv-provisioner/pkg/version.Version=${VERSION}" \
```
Comment (akhilerm) [inline buildscripts/build.sh:108]: @mynktl This was the change that I suggested we should do in NFS also.

**Comment by niladrih** [comment]
Related to https://github.com/openebs/dynamic-nfs-provisioner/pull/45

**Review by akhilerm — APPROVED** [review]
The changes look good.

Have put up a question / suggestion in the comment section

**Review by mynktl — APPROVED** [review]

**Pull request merged**